### PR TITLE
Correct consecutive get()  in cypress instruction #2712

### PR DESCRIPTION
**Pull request by JacquesBonet**
- Create a new svg access function permitting to acces the right element.
- correct another problem about headroom css class. This class doesn't exist with a production build using react-script, so use alternative css class.

### CODE REVIEW

File: cypress/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "name": "e2e",
     "version": "0.1.0",
     "scripts": {
+        "serve": "node ./infinitStart.js",
```
Comment (djhi) [inline cypress/package.json]: Why is this needed ? Looks irrelevant to this PR

Comment (djhi) [reply cypress/package.json]: Are you aware of the npm commands inside the root `package.json`. If you can't use make, you can:

launch a terminal and start `yarn run-simple`

launch another terminal and start `yarn test-e2e-local`

Can you remove the new ones?

Comment (JacquesBonet) [reply cypress/package.json]: sure
